src/dq_suite/output_transformations.py

```diff
@@ -33,16 +36,30 @@ def list_of_dicts_to_df(
     return spark_session.createDataFrame((Row(**x) for x in list_of_dicts), schema=schema)
 
 
+def construct_regel_id(
+    df: str,
+    output_columns_list: list,
+) -> DataFrame:
+    df_with_id = df.withColumn("regelId", xxhash64(col("regelNaam"), col("regelParameters"), col("bronTabelId")))
+    return df_with_id.select(*output_columns_list)
+    
+
+def create_parameter_list_from_results(result: dict) -> list:
```

Comment:
Is het mogelijk om aan te geven wat de precieze return type is? Is het een list van strings, of kan `parameters` alles zijn?